templates/csharp/link-unfurling/Properties/launchSettings.json

```diff
@@ -11,6 +11,17 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "hotReloadProfile": "aspnetcore"
+    },
+    "Outlook (browser)": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "https://outlook.office.com/mail?login_hint=%USERNAME%",
```

Comment:
I am curious about this side loading url isn't included the app info, how to try it in user experience?

Reply:
The tab app uses link "https://outlook.office.com/host/${M365_APP_ID}" to side load, this link would open a web page of the app. Since message extension doesn't have a web page, using this link would result in error.

To use message extension in Outlook, you need to click "New email" first. For link unfurling, just paste the link into email area; for search, click "Apps" button and find your search app.